pkg/pwgen/xkcdgen/pwgen.go

```diff
@@ -16,20 +16,21 @@ func Random() string {
 // RandomLength returns a random passphrase combined from the desired number.
 // of words. Words are drawn from lang.
 func RandomLength(length int, lang string) (string, error) {
-	return RandomLengthDelim(length, " ", lang)
+	return RandomLengthDelim(length, " ", lang, false)
 }
 
 // RandomLengthDelim returns a random passphrase combined from the desired number
 // of words and the given delimiter. Words are drawn from lang.
-func RandomLengthDelim(length int, delim, lang string) (string, error) {
+func RandomLengthDelim(length int, delim, lang string, capitalize, numbers bool) (string, error) {
 	g := xkcdpwgen.NewGenerator()
 	g.SetNumWords(length)
 	g.SetDelimiter(delim)
-	g.SetCapitalize(delim == "")
+	g.SetCapitalize(delim == "" || capitalize)
```

Comment:
TIL that we already supported the feature when we set the delimiter the the empty string, nice.